src/storageClients/BlobStorageManager.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const Promise = require('bluebird');
+const azure = Promise.promisifyAll(require('azure-storage'));
+
+const BLOB_STORAGE_CONNECTION_STRING = process.env.BLOB_STORAGE_CONNECTION_STRING;
+
+module.exports = {
+
+  Get: function(containerName, blobName, id) {
+    let blobSvc = azure.createBlobService(BLOB_STORAGE_CONNECTION_STRING);
+    return new Promise((resolve, reject) => {
+      blobSvc.getBlobToTextAsync(containerName, blobName, null)
+        .then(text => {
+          let item = JSON.parse(text)[id];
+          if(item) {
+            resolve(JSON.parse(text)[id]);
+          } else {
+            resolve(null);
+          }
+        })
+        .catch(err => {
+          reject(err);
+        });
+    });
+  },
+
+  List: function(containerName, blobNames) {
+    return new Promise((resolve, reject) => {
+      Promise.all(getListPromises(containerName, blobNames))
+        .then(itemsArrays => {
+          let mergedItems = [].concat.apply([], itemsArrays);
```

Comment:
ah yea that did cross my mind and then I forgot to use it!! that would make more sense!